operate/schema/src/main/java/io/camunda/operate/schema/elasticsearch/ElasticsearchSchemaManager.java

```diff
@@ -92,6 +92,22 @@ public void createSchema() {
     createIndices();
   }
 
+  @Override
+  public void checkAndUpdateIndices() {
+    indexDescriptors.forEach(
```

Comment:
I think you missed the other indices which can be find under 

`List<TemplateDescriptor> templateDescriptors;`

If you debug or use logs like these in `SchemaManager`s
```java
final var added =
                setIndexSettingsFor(indexSettings, descriptor.getDerivedIndexNamePattern());
            LOGGER.info("Add ILM for {}: {}", descriptor.getDerivedIndexNamePattern(), added);
```
and for delete
```java
 final var deleted =
                retryElasticsearchClient.deleteIndexPolicyFor(
                    descriptor.getDerivedIndexNamePattern());
            LOGGER.info("Deleted ILM for {}: {}", descriptor.getDerivedIndexNamePattern(), deleted);
```
you can see which indices are affected.